xml/System.IO/FileStream.xml

```diff
@@ -4319,9 +4321,6 @@ If the operation is canceled before it completes, the returned task contains the
         <exception cref="T:System.ArgumentOutOfRangeException">
           <paramref name="offset" /> or <paramref name="count" /> is negative.</exception>
         <exception cref="T:System.ArgumentException">The sum of <paramref name="offset" /> and <paramref name="count" /> is larger than the buffer length.</exception>
-        <exception cref="T:System.NotSupportedException">The stream does not support writing.</exception>
```

Comment:
I deleted these exceptions because the guidance is that stored exceptions shouldn't be documented as regular (synchronous) exceptions, since the async method still completes and returns something.